### PR TITLE
Fix stringtie and skip old builds

### PR DESCRIPTION
**Pull request by kyleabeauchamp**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/stringtie/1.0.3/meta.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ about:
   summary: Transcriptome assembly and quantification for RNA-seq
 build:
   number: 1
-  skip: True # [osx]
+  skip: True  # See build issues in https://github.com/bioconda/bioconda-recipes/pull/9460/
```
Comment (bgruening) [inline recipes/stringtie/1.0.3/meta.yaml]: can you put those recipes into your blacklist?

Comment (kyleabeauchamp) [reply recipes/stringtie/1.0.3/meta.yaml]: Where do you want me to list them, specifically?

Comment (bgruening) [reply recipes/stringtie/1.0.3/meta.yaml]: build-fail-blacklist?

**Comment by kyleabeauchamp** [comment]
Deprecated in favor of https://github.com/bioconda/bioconda-recipes/pull/9472

**Pull request closed without merging**